.gitignore

```diff
@@ -12,3 +12,4 @@ package-lock.json
 /tmp
 /coverage
 /.nyc_output
+/.idea
```

Comment:
Assume this is an artefact of your IDE?  If so, could you group them under a comment just so it's clear? e.g...

# VS Code
/.vs_code